app/models/task.rb

```diff
@@ -0,0 +1,33 @@
+# == Schema Information
+#
+# Table name: tasks
+#
+#  project_id  :string
+#  name        :string
+#  description :text
+#  state       :integer          default(10)
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  id          :uuid             not null, primary key
+#
+
+class Task < ActiveRecord::Base
+  belongs_to :project
+  validates :name, presence: true
+  validates :state, presence: true
+  validates :project_id, presence: true
+  after_commit :send_message, on: :update
```

Comment:
Nice. The decorator is a better approach too, because you could add logic to prevent duplicate text messages if/when the object is ever updated again.